cylc/flow/network/server.py

```diff
@@ -99,7 +99,7 @@ class SuiteRuntimeServer(ZMQSocketBase):
 
     """
 
-    API = 4  # cylc API version
+    API = API  # cylc API version
```

Comment:
This line can be dispensed with :smile:!

Reply:
Well it could be read as `self.API = API`, I really just left it in for the end point `api`.

Reply:
Fixed - I thought an endpoint read the class attributes... Not the case.